client/consensus/common/src/import_queue/buffered_link.rs

```diff
@@ -52,8 +52,10 @@ use super::BlockImportResult;
 /// Wraps around an unbounded channel from the `futures` crate. The sender implements `Link` and
 /// can be used to buffer commands, and the receiver can be used to poll said commands and transfer
 /// them to another link.
```

Comment:
also can we add something to doc to reflect new argument?

Reply:
Done